src/resolverHelpers.js

```diff
@@ -1,60 +1,77 @@
 const request = require('request-promise');
 
-const { graphRoot }  = require('./constants');
+const { graphRoot } = require('./constants');
 
-function makeRequest(path, authorization, isSecondary) {
+function makeRequest(path, authorization, args, isSecondary) {
+  if (args.id != null) {
+    url = url + '/' + args.id;
```

Comment:
This causes the following error:
```
{
  "errors": [
    {
      "message": "url is not defined",
      "locations": [
        {
          "line": 28,
          "column": 2
        }
      ],
      "path": [
        "users"
      ]
    }
  ],
  "data": {
    "users": null
  }
}
```

Reply:
Yes - thanks for catching this. I made a minor change with url/path just before submitting the PR.